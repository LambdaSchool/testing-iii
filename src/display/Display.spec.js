// Test away!
import React from 'react';
import {render } from '@testing-library/react'

import Display from './Display'

test('Let it display', ()=>{
    const closed= true; 
    const locked= true;
    const tree = render(<Display closed={closed} locked={locked}/>)
    expect(tree.baseElement).toMatchSnapshot();

} )