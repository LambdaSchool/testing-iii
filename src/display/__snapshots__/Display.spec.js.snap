// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Let it display 1`] = `
<body>
  <div>
    <div
      class="display panel"
    >
      <div
        class="led red-led"
      >
        Locked
      </div>
      <div
        class="led red-led"
      >
        Closed
      </div>
    </div>
  </div>
</body>
`;
